docs/tutorials/01_ui/viz_ui_slider.py

```diff
@@ -97,10 +137,10 @@ def translate_cube(slider):
 show_manager.scene.set_camera(position=(0, 0, 150))
 show_manager.scene.reset_clipping_range()
 show_manager.scene.azimuth(30)
-interactive = False
+interactive = True
```

Comment:
Can you put it back to `False` ? thank you

Reply:
got it.